### ISSUE TITLE
src/loragw_spi.native.c:301: error: 'struct spi_ioc_transfer' has no member named 'tx_buf'

### ISSUE THREAD

**Issue by zhoubx123** [Lora-net/lora_gateway#81]
compile error,who can help me? I have use redhat linux 6, and when I compile 

/opt/arm-linux-gcc/4.3.3/bin/../arm-none-linux-gnueabi/libc/usr/include/linux/spi/spidev.h:83: error: expected specifier-qualifier-list before '__u64'
src/loragw_spi.native.c: In function 'lgw_spi_open':
src/loragw_spi.native.c:91: error: '__u8' undeclared (first use in this function)
src/loragw_spi.native.c:91: error: (Each undeclared identifier is reported only once
src/loragw_spi.native.c:91: error: for each function it appears in.)
src/loragw_spi.native.c:102: error: '__u32' undeclared (first use in this function)
src/loragw_spi.native.c: In function 'lgw_spi_w':
src/loragw_spi.native.c:195: error: 'struct spi_ioc_transfer' has no member named 'tx_buf'
src/loragw_spi.native.c:196: error: 'struct spi_ioc_transfer' has no member named 'len'
src/loragw_spi.native.c:197: error: 'struct spi_ioc_transfer' has no member named 'speed_hz'
src/loragw_spi.native.c:198: error: 'struct spi_ioc_transfer' has no member named 'cs_change'
src/loragw_spi.native.c:199: error: 'struct spi_ioc_transfer' has no member named 'bits_per_word'
src/loragw_spi.native.c:203: error: 'struct spi_ioc_transfer' has no member named 'len'
src/loragw_spi.native.c: In function 'lgw_spi_r':
src/loragw_spi.native.c:246: error: 'struct spi_ioc_transfer' has no member named 'tx_buf'
src/loragw_spi.native.c:247: error: 'struct spi_ioc_transfer' has no member named 'rx_buf'
src/loragw_spi.native.c:248: error: 'struct spi_ioc_transfer' has no member named 'len'
src/loragw_spi.native.c:249: error: 'struct spi_ioc_transfer' has no member named 'cs_change'
src/loragw_spi.native.c:253: error: 'struct spi_ioc_transfer' has no member named 'len'
src/loragw_spi.native.c: In function 'lgw_spi_wb':
src/loragw_spi.native.c:301: error: 'struct spi_ioc_transfer' has no member named 'tx_buf'
src/loragw_spi.native.c:302: error: 'struct spi_ioc_transfer' has no member named 'len'
src/loragw_spi.native.c:303: error: 'struct spi_ioc_transfer' has no member named 'cs_change'
src/loragw_spi.native.c:304: error: 'struct spi_ioc_transfer' has no member named 'cs_change'
src/loragw_spi.native.c:308: error: 'struct spi_ioc_transfer' has no member named 'tx_buf'
src/loragw_spi.native.c:309: error: 'struct spi_ioc_transfer' has no member named 'len'
src/loragw_spi.native.c:310: error: 'struct spi_ioc_transfer' has no member named 'len'
src/loragw_spi.native.c: In function 'lgw_spi_rb':
src/loragw_spi.native.c:363: error: 'struct spi_ioc_transfer' has no member named 'tx_buf'
src/loragw_spi.native.c:364: error: 'struct spi_ioc_transfer' has no member named 'len'
src/loragw_spi.native.c:365: error: 'struct spi_ioc_transfer' has no member named 'cs_change'
src/loragw_spi.native.c:366: error: 'struct spi_ioc_transfer' has no member named 'cs_change'
src/loragw_spi.native.c:370: error: 'struct spi_ioc_transfer' has no member named 'rx_buf'
src/loragw_spi.native.c:371: error: 'struct spi_ioc_transfer' has no member named 'len'
src/loragw_spi.native.c:372: error: 'struct spi_ioc_transfer' has no member named 'len'
make[1]: *** [obj/loragw_spi.o] Error 1
make[1]: Leaving directory `/home/work/gateway-new/lora_gateway/libloragw'
make: *** [all] Error 2

**Comment by mcoracin** [comment]
This is a build environment issue, out of scope of this project.
